### ISSUE TITLE
G920 Forza horizon 4 Force Feedback delays. Tried command in steam game launch options, but it prevents game from launching.

### ISSUE THREAD

**Issue by roadburn9966** [berarma/ffbtools#41]
`/home/username/ffbtools/bin/ffbwrap --throttling --throttling-time 19 /dev/input/by-id/usb-Logitech_G920_Driving_Force_Racing_Wheel_for_Xbox_One_0000f9b5d17d44f1-event-joystick -- %command%`

This is the command I used in the steam launch options, but then the game just doesn't launch. If I run the command in the terminal, it says `bash: syntax error near unexpected token newline`

Thank you for the help. I'm also new to linux so if there's a clear error I made, that's why.

**Comment by PoorPocketsMcNewHold** [comment]
Having the same issue as well and willing to provide logs.

`/home/pm/Documents/ffbtools/bin/ffbwrap --update-fix --logger=/home/pm/fh4FFB.log --throttling --throttling-time=15 /dev/input/by-id/usb-Logitech_G920_Driving_Force_Racing_Wheel_for_Xbox_One_0000aa264efcff1f-event-joystick -- %command%`
[fh4FFB.log-20240916174454.log](https://github.com/user-attachments/files/17016053/fh4FFB.log-20240916174454.log)

Did you compile the tools? In addition, is your username… "username"? Check if the path is correct, it depends on where you put the executable.

**Comment by roadburn9966** [comment]
I think I compiled the tools, in the bin folder I see a "ffbwrap" and "ffbplay" file. How would I compile the tools? Is it by running `make` in the ffbtools folder where the "Makefile" is? If so, when I do that it says `make: Nothing to be done for 'all'.` 

Also I put "username" in the post because I didn't want to give out my real name.

**Comment by berarma** [comment]
@roadburn9966: Running the same command in the terminal won't work, Steam replaces `%command%` with the game executable. Make says there's nothing to do because the binaries are already built.

@PoorPocketsMcNewHold: How are you having the same issue and providing a log produced by another game running through ffbtools?

**Comment by PoorPocketsMcNewHold** [comment]
> @roadburn9966: Running the same command in the terminal won't work, Steam replaces `%command%` with the game executable. Make says there's nothing to do because the binaries are already built.
> 
> @PoorPocketsMcNewHold: How are you having the same issue and providing a log produced by a game running through ffbtools?

> __G920 Forza horizon 4 Force Feedback delays.__ Tried command in steam game launch options, but it prevents game from launching. 

Perhaps it needs its own issue in that case.

**Comment by berarma** [comment]
> > **G920 Forza horizon 4 Force Feedback delays.** Tried command in steam game launch options, but it prevents game from launching.
> 
> Perhaps it needs its own issue in that case.

Yes, please. You took the context as the issue. The issue here is ffbtools not running.
